src/docs/overview/best-practices.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Best Practices
+---
+
+Railway is a highly versatile platform, offering various ways to use it, though some may be less optimal than others for most use cases. the topics here aim to help you maximize the platform's potential.
+
+## Use Private networking
+
+[Private networking](/reference/private-networking) allows services within a project to communicate internally without the need to expose them publicly.
+
+The private network has no egress fees, allowing you to avoid service-to-service egress costs when communicating with a database or other services within the same project.
+
+## Deploying related Services into the same Project
+
+It can be all too easy to create a new project each time you deploy a database or service, but there are a few reasons to keep related services within the same project.
+
+- **Private networking** - The private network is scoped to a single environment within a project, having all related services within a single project will allow you to use private networking for faster networking along with no egress fees for service-to-service communication.
+
+- **Project clutter** - Deploying a new service or database as an entire project will quickly become overwhelming and clutter your dashboard.
+
+## Use Reference variables
+
```

Comment:
maybe add some words about how reference variables make your variable assignments dynamic. instead of hardcoding your variable values, use reference variables so that they update when the value changes.

Reply:
I have, just a few more lines down 😄

> Using reference variables ensures your variable values are always in sync. Change your TCP proxy? The variable updates. Change your backend hostname? The variable updates.

Reply:
Ah okay hmm .. I feel like we can make things more clear..  Now that I think about it, this applies for each section actually.

I think you should rework them a bit so they all follow a similar pattern.. start off with the scenario (_when doing X_) then the directive (_do Y_) then state why (_because Z_), then show what it looks like and point to the appropriate docs page.  e.g. 

"When connecting your services in Railway, always use the private network.  Communication is much faster and you will avoid egress fees" ... then your graphic and link to the private network docs.

whatcha think?

Reply:
Yeah that makes sense, might be better than just being told to do something!

Reply:
I think it's far easier to read now that each topic follows the same pattern!

Reply:
Looks great @brody192 ❤️  (sorry if you were waiting on me)